### PR TITLE
Style Guide

### PR DESCRIPTION
**Pull request by meshula**
Starting a PR to revise the style-guide in order to be able to take advantage of the suggest tag and other code review tools here on github.

### CODE REVIEW

**Comment by meshula** [comment]
@KarthikRIyer Hi Karthik! You're welcome to participate and contribute on a style guide over here on this PR!

File: docs/style-guide.md [hunk 1]
```diff
@@ -101,7 +107,11 @@ namespace opentime
 
 # Environment Variables
 
-No runtime behavior of OpenTimelineIO is modified by the settings of environment variables.
+No runtime behavior of the OpenTimelineIO C++ runtime is modified by the settings of environment variables. This is because the C++ libraries must run in environments without environment variables, or that are sandboxed for security.
+
+The Python based tools are configured through two variables: OTIO_PLUGIN_MANIFEST_PATH (describe effect here), and OTIO_DEFAULT_+MEDIA_LINKER (describe effect here).
```
Comment (meshula) [inline docs/style-guide.md:112]:
```suggestion
```

File: docs/style-guide.md [hunk 2]
```diff
@@ -101,7 +107,11 @@ namespace opentime
...
+
+The tests invoked as part of setup.py can be bypased by creating a variable named OTIO_DISABLE_SHELLOUT_TESTS
```
Comment (meshula) [inline docs/style-guide.md:113]:
```suggestion
```

File: docs/style-guide.md [hunk 3]
```diff
@@ -101,7 +107,10 @@ namespace opentime
 
 # Environment Variables
 
-No runtime behavior of OpenTimelineIO is modified by the settings of environment variables.
+No runtime behavior of the OpenTimelineIO C++ runtime is modified by the settings of environment variables. This is because the C++ libraries must run in environments without environment variables, or that are sandboxed for security.
+
+
+The tests invoked as part of setup.py can be bypased by creating a variable named OTIO_DISABLE_SHELLOUT_TESTS
```
Comment (meshula) [inline docs/style-guide.md]:
```suggestion
```

**Review by reinecke — APPROVED** [review]

**Pull request merged**